book/src/drivers.md

```diff
@@ -73,6 +73,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Intelliflash Block Storage](https://github.com/DDNStorage/intelliflash-csi-block-driver) | `intelliflash-csi-block-driver.intelliflash.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Intelliflash  Block Storage | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
 [Intelliflash File Storage](https://github.com/DDNStorage/intelliflash-csi-file-driver) | `intelliflash-csi-file-driver.intelliflash.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Intelliflash  File Storage | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
 [ionir ](https://github.com/ionir-cloud) | `ionir` | v1.2 | A Container Storage Interface (CSI) Driver for [ionir](https://www.ionir.com/) Kubernetes-Native Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Cloning
+[JD Cloud Storage Platform Block](https://github.com/jd-opensource/jdcsp-csi/) | `jdcsp-block.csi.jdcloud.com` | v1.0 | A Container Storage Interface (CSI) Driver for JD-CSP Block | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
```

Comment:
Column 3 is for supported CSI spec version, not CSI driver version.  Are you sure the highest CSI spec version your CSI driver can support is v1.0?

Reply:
Hi

The driver compatible highest CSI spec version is v1.8.0.  Already fixed it.

Thanks